nora/src/content/browser-splitView/browser-splitView.tsx

```diff
@@ -0,0 +1,213 @@
+import { insert } from "@solid-xul/solid-xul";
+import splitViewStyle from "./browser-splitView.pcss?inline";
+import { HideSplitViewSplitter, SplitViewSplitter } from "./split-view-splitter";
+
+export namespace gSplitView {
+  let StyleElement = () => {
+    return <style id="splitViewCSS">{ splitViewStyle }</style>
+  }
+
+  export function setSplitView(tab: {
```

Comment:
use `type` keyword to make complex object type
it is good in separating and making less in func arguments

Reply:
Done!